File: Solutions/1347. Minimum Number of Steps to Make Two Strings Anagram/1347. Minimum Number of Steps to Make Two Strings Anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        int res=0;
        int cnt[26]={};
        for(const auto& c:s)
            ++cnt[c-'a'];
        for(const auto& c:t)
            --cnt[c-'a'];
        for(int i=0;i<26;++i)
            res+=cnt[i]==0?0:abs(cnt[i]);
        return res/2;
    }
};

File: Solutions/1347. Minimum Number of Steps to Make Two Strings Anagram/1347. Minimum Number of Steps to Make Two Strings Anagram.py
class Solution:
    def minSteps(self, s: str, t: str) -> int:
        res,cnt=0,[0]*26
        for c in s:
            cnt[ord(c)-ord('a')]+=1
        for c in t:
            cnt[ord(c)-ord('a')]-=1
        for i in range(26):
            x=0 if cnt[i]==0 else abs(cnt[i])
            res+=x
        return res//2